src/Tizen.Network.NetworkMonitor/Tizen.Network.NetworkMonitor/UrlChecker.cs

```diff
@@ -0,0 +1,116 @@
+/*
+ * Copyright (c) 2019 Samsung Electronics Co., Ltd All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the License);
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an AS IS BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+using System;
+using System.Collections.Generic;
+
+namespace Tizen.Network.NetworkMonitor
+{
+    /// <summary>
+    /// A class which checks whether given URLs are reachable or not.
+    /// </summary>
+    /// <since_tizen> 6 </since_tizen>
+    public static class UrlChecker
```

Comment:
Why do you provide is API?
It already can implement with System.Net API

Reply:
The purpose of this API is to check if the device uses the given URLs or not. It is used for Internet access check.
Do you mean System.Net.WebRequest API? I'll check it.
Thanks.

Reply:
Yes, We have enough way to check URL reachable. 
We not need a new API to check URL

https://www.google.co.kr/search?newwindow=1&source=hp&ei=y0pJXfL0I5KvmAXPmqyoDg&q=c%23+url+reachable+test&oq=c%23+url+reachable+test&gs_l=psy-ab.3..33i160.73.16762..17475...6.0..1.264.3787.0j22j2......0....1..gws-wiz.....0..0j0i131j35i39j0i203j0i8i30j33i21j0i8i13i30.8QLPwXh7h54&ved=0ahUKEwiyhqTM-e3jAhWSF6YKHU8NC-UQ4dUDCAU&uact=5